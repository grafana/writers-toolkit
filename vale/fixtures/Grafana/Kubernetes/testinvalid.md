Don't use:

- cron job
- deamonset
- Kubernetes deployment
- pod
- replicaset
- statefulset
- kubelet
- kubectl
- Kubectl
- Kubelet
