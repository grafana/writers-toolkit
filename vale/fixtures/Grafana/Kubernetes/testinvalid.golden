testinvalid.md:3:3:Grafana.Kubernetes:Use 'CronJob' instead of 'cron job'.
testinvalid.md:4:3:Grafana.Kubernetes:Use 'DaemonSet' instead of 'deamonset'.
testinvalid.md:5:3:Grafana.Kubernetes:Use 'Kubernetes Deployment' instead of 'Kubernetes deployment'.
testinvalid.md:6:3:Grafana.Kubernetes:Use 'Pod' instead of 'pod'.
testinvalid.md:7:3:Grafana.Kubernetes:Use 'ReplicaSet' instead of 'replicaset'.
testinvalid.md:8:3:Grafana.Kubernetes:Use 'StatefulSet' instead of 'statefulset'.
testinvalid.md:9:3:Grafana.Kubernetes:Use '`kubelet`' instead of 'kubelet'.
testinvalid.md:10:3:Grafana.Kubernetes:Use '`kubectl`' instead of 'kubectl'.
testinvalid.md:11:3:Grafana.Kubernetes:Use '`kubectl`' instead of 'Kubectl'.
testinvalid.md:12:3:Grafana.Kubernetes:Use '`kubelet`' instead of 'Kubelet'.
